docs/homelab/headscale/per-os/macos-appstore.md

```
@@ -0,0 +1,107 @@
+# Verify.
+defaults read /Library/Preferences/io.tailscale.ipn.macos ControlURL
+# Expected: https://hs.lab.hole-truth.org
+```
+
+This tells the App Store build that the next login attempt should go to Headscale, not Tailscale Inc.
+
+### Step 5: Log in via the menu bar
+
+Click the Tailscale icon in the menu bar → **Log in**. This opens a browser tab pointing at `https://hs.lab.hole-truth.org/register/<key>` (or similar). The browser tab will show a Headscale registration prompt — typically a CLI command of the form:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)